### ISSUE TITLE
only integer tensors of a single element can be converted to an index

### ISSUE THREAD

**Issue by ShangYuming** [JayParks/transformer#4]
When I run the colder, there is an error  in model.py (line 53):

`self.layers = nn.ModuleList(
            [self.layer_type(d_k, d_v, d_model, d_ff, n_heads, dropout) for _ in range(n_layers)])`

only integer tensors of a single element can be converted to an index

How can I fix it
